crawl-ref/source/tiledgnbuf.cc

```diff
@@ -223,6 +224,9 @@ void DungeonCellBuffer::draw()
 void DungeonCellBuffer::add_blood_overlay(int x, int y, const packed_cell &cell,
                                           bool is_wall)
 {
+    if (!Options.show_blood)
+        return;
+
     if (cell.is_liquefied && !is_wall)
```

Comment:
I'm pretty sure this block is for Leda's Liquefaction, not blood, so the options check should maybe be in the `else if` condition below

Reply:
I will fix that.